htdocs/manage/settings/index.bml

```diff
@@ -32,8 +32,16 @@ body<=
     my $authas;
     my $u;
 
-    if ($remote) {
-        $authas = $GET{authas} || $remote->user;
+    $authas = $GET{authas} || $remote->user if $remote;
+
+    my $can_view_other = ( $remote && $remote->has_priv( "canview", "subscriptions" ) );
```

Comment:
I went back and forth on whether to use the `? 1 : 0` construction at the end of that, and ultimately left it off.